From: Jason Rhinelander <jason@imaginary.ca>
Date: Sun, 5 May 2019 12:40:15 -0300
Subject: Allow customized version tag

---
 cmake/Version.cmake | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/cmake/Version.cmake b/cmake/Version.cmake
index 7d7a927..fb124fd 100644
--- a/cmake/Version.cmake
+++ b/cmake/Version.cmake
@@ -32,7 +32,10 @@ function (write_static_version_header hash)
 endfunction ()
 
 find_package(Git QUIET)
-if (GIT_FOUND OR Git_FOUND)
+if (GYUANX_VERSION_TAG)
+  # Custom version tag (used by debian packaging)
+  write_static_version_header("${GYUANX_VERSION_TAG}")
+elseif (GIT_FOUND OR Git_FOUND)
   message(STATUS "Found Git: ${GIT_EXECUTABLE}")
   set(VERSIONTAG "@VERSIONTAG@") # Will be replaced again by GenVersion.cmake, below.
   configure_file("${CMAKE_SOURCE_DIR}/src/version.cpp.in" "${CMAKE_BINARY_DIR}/version.cpp.in")
