From: Jason Rhinelander <jason@imaginary.ca>
Date: Sun, 14 Jul 2019 18:11:41 -0300
Subject: Version and install librandomx

---
 external/CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/external/CMakeLists.txt b/external/CMakeLists.txt
index effcea3..a247bef 100644
--- a/external/CMakeLists.txt
+++ b/external/CMakeLists.txt
@@ -95,6 +95,12 @@ endif()
 add_subdirectory(randomx EXCLUDE_FROM_ALL)
 add_subdirectory(loki-mq)
 
+set_target_properties(randomx PROPERTIES
+  VERSION "${LOKI_VERSION_MAJOR}.${LOKI_VERSION_MINOR}.${LOKI_VERSION_PATCH}")
+install(TARGETS randomx
+  ARCHIVE DESTINATION lib${LOKI_INSTALL_LIBDIR_SUFFIX}
+  LIBRARY DESTINATION lib${LOKI_INSTALL_LIBDIR_SUFFIX})
+
 install(
   DIRECTORY boost
   DESTINATION include${LOKI_INSTALL_INCLUDEDIR_SUFFIX}
