From: Jason Rhinelander <jason@imaginary.ca>
Date: Fri, 13 Mar 2020 01:29:19 -0300
Subject: Fix missing dep

---
 src/cryptonote_core/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/cryptonote_core/CMakeLists.txt b/src/cryptonote_core/CMakeLists.txt
index 89c96ba..4830741 100644
--- a/src/cryptonote_core/CMakeLists.txt
+++ b/src/cryptonote_core/CMakeLists.txt
@@ -51,6 +51,7 @@ target_link_libraries(cryptonote_core
     ringct
     device
     sqlite3
+    checkpoints
   PRIVATE
     Boost::program_options
     Boost::chrono
