From: Jason Rhinelander <jason@imaginary.ca>
Date: Wed, 9 Oct 2019 21:03:51 -0300
Subject: dumb ARM_ID/ARCH logic workaround

---
 CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 283479f..0a46d4b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -190,7 +190,9 @@ else()
   set(ARCH_ID "${ARCH}")
 endif()
 endif()
-string(TOLOWER "${ARCH_ID}" ARM_ID)
+if (NOT ARM_ID)
+  string(TOLOWER "${ARCH_ID}" ARM_ID)
+endif()
 string(SUBSTRING "${ARM_ID}" 0 3 ARM_TEST)
 if (ARM_TEST STREQUAL "arm")
   set(ARM 1)
